{Release} Upgrade to Azure CLI 2.64.0 (#29797)
Azure/azure-cli: src/azure-cli/HISTORY.rst

Release History
===============

2.64.0
++++++

**ACR**

> * Improve deprecation message for `az acr helm` (#29700)

Reviewer: ```suggestion
* `az acr helm`: Improve deprecation message (#29700)
```

> * Improve deprecation message for `az acr helm` (#29700)

Reviewer 2: ```suggestion
* `az acr helm`: Improve deprecation message  (#29700)
```

> * Improve deprecation message for `az acr helm` (#29700)

Reviewer 3: ```suggestion
* `az acr helm`: Improve deprecation message (#29700)
```

**AKS**

* `az aks nodepool update`: Add `--enable/disable-fips-image` flags for GA mutable fips (#29695)
* `az aks create/update`: Support UserAssigned Managed Identity for grafana linking in managed prometheus (#29713)
* `az aks create/update`: Update the VM SKU regex validation to include larger set of VMs for Azure Container Storage (#29726)

**App Service**

> * `az staticwebapp create` and `az staticwebapp update`: Add `Dedicated` as supported SKU (#29514)

Reviewer 3: ```suggestion
* `az staticwebapp create/update`: Add `Dedicated` as supported SKU (#29514)
```

> * `az staticwebapp create` and `az staticwebapp update`: Add `Dedicated` as supported SKU (#29514)

Reviewer: ```suggestion
* `az staticwebapp create/update`: Add `Dedicated` as supported SKU (#29514)
```

* `az webapp config access-restriction add/remove`: Allow skipping service tag validation (#29710)

**ARM**

* `az ts create`: Fix incorrect handling of whitespace in string values (#29623)

**ARO**

* `az aro create`: Add feature for adding Multiple Public IPs on cluster load balancer (#29693)
* `az aro update`: Add feature for adding/removing Multiple Public IPs on cluster load balancer (#29693)

**Compute**

* `az vm update`: Add support of Gen1 VM to trusted launch upgrade (#29655)
* `az capacity reservation`: GA command group (#29775)

**Container app**

* Fix #28998: `az containerapp env workload-profile add/update`: Fix `NoneType` object is not iterable error when environment doesn't enable workload profile (#29682)
* `az containerapp create/up`: Make `--target-port` optional (#29702)
* `az containerapp env create`: Deprecate unused argument `--docker-bridge-cidr` (#29746)
* `az containerapp job stop`: Deprecate option to stop list of given job executions (#29728)
* Fix #29711: `az containerapp logs show`: Fix JSON escaping by default or with `--format json` (#29767)

**HDInsight**

* `az hdinsight create`: Support setting IP tags when creating HDInsight cluster (#29752)

> * `az hdinsight update`: Support update the managed identity of the cluster (#29752)
> * `az hdinsight azure-monitor-agent show`: Get update the managed identity of the cluster (#29752)

Reviewer 2: ```suggestion
* `az hdinsight update`: Support updating the managed identity of the cluster (#29752)
* `az hdinsight azure-monitor-agent show`: Support getting the managed identity of the cluster (#29752)
```

* `az hdinsight azure-monitor-agent enable`: Enable the Azure Monitor Agent logs integration on an HDInsight cluster (#29752)
* `az hdinsight azure-monitor-agent disable`: Disable the Azure Monitor Agent logs integration on an HDInsight cluster (#29752)

**MySQL**

* `az mysql flexible-server maintenance`: New command group for managing maintenance of MySQL flexible server (#29505)
* `az mysql flexible-server update`: Remove storage passing for server update (#29739)

**NetAppFiles**

> * Update max for usageThreshold (#29624)

Reviewer 2: ```suggestion
* `az netappfiles volume create/update`: Update max for usageThreshold (#29624)
```

> * Update max for usageThreshold (#29624)

Reviewer 4: ```suggestion
* `az netappfiles volume create/update`: Update max for `--usage-threshold` (#29624)
```

> * Update max for usageThreshold (#29624)

Reviewer 3: ```suggestion
* `az netappfiles volume create/update`: Update max quota for `--usage-threshold` (#29624)
```
